x-pack/plugins/timelines/public/components/t_grid/helpers.tsx

```diff
@@ -276,3 +291,57 @@ export const getDefaultViewSelection = ({
 
 /** This local storage key stores the `Grid / Event rendered view` selection */
 export const ALERTS_TABLE_VIEW_SELECTION_KEY = 'securitySolution.alerts.table.view-selection';
+
+export const buildISQueryMatch = ({
```

Comment:
consider applying the casing feedback in [this comment](https://github.com/elastic/kibana/pull/144988#discussion_r1023188666) here, and to the other functions (`buildEXISTSQueryMatch` and `buildISONEOFQueryMatch`) in this file

Reply:
done!